sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StandardMetricTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.Exporter.Models;
+using OpenTelemetry.Metrics;
+using OpenTelemetry;
+using Xunit;
+using Azure.Monitor.OpenTelemetry.Exporter.Tests.CommonTestFramework;
+using OpenTelemetry.Trace;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
+{
+    public class StandardMetricTests
+    {
+        [Fact]
+        public void ValidateRequestDurationMetric()
+        {
+            var activitySource = new ActivitySource(nameof(StandardMetricTests.ValidateRequestDurationMetric));
+            var traceTelemetryItems = new ConcurrentBag<TelemetryItem>();
+            var metricTelemetryItems = new ConcurrentBag<TelemetryItem>();
+            var exporterOptions = new AzureMonitorExporterOptions();
+            exporterOptions.ConnectionString = "InstrumentationKey=Ikey;IngestionEndpoint=https://westus2-0.in.applicationinsights.azure.com/;LiveEndpoint=https://westus2.livediagnostics.monitor.azure.com/";
+
+            var metricReader = new PeriodicExportingMetricReader(new AzureMonitorMetricExporter(new MockTransmitter(metricTelemetryItems)))
+            { TemporalityPreference = MetricReaderTemporalityPreference.Delta };
+
+            var standardMetricCustomProcessor = new StandardMetricsExtractionProcessor(metricReader);
+
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .SetSampler(new AlwaysOnSampler())
+                .AddSource(nameof(StandardMetricTests.ValidateRequestDurationMetric))
+                .AddProcessor(standardMetricCustomProcessor)
+                .AddProcessor(new BatchActivityExportProcessor(new AzureMonitorTraceExporter(new MockTransmitter(traceTelemetryItems))))
+                .Build();
+
+            using (var activity = activitySource.StartActivity("Test", ActivityKind.Server))
+            {
+                activity?.SetTag(SemanticConventions.AttributeHttpStatusCode, 200);
+            }
+
+            tracerProvider?.ForceFlush();
+            standardMetricCustomProcessor?.StandardMetricMeterProvider?.ForceFlush();
+
+            Assert.Single(traceTelemetryItems);
```

Comment:
exporting Activity in a unit test can be flaky. you should use a SpinWait pattern here.
For example: https://github.com/Azure/azure-sdk-for-net/blob/4c575251241724be53423b43414842beed9695f7/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests/TestFramework/WebApplicationTestsBase.cs#L37-L48